root/static/scripts/edit/externalLinks.js

```diff
@@ -549,6 +549,12 @@ export class ExternalLinksEditor
                        {example_url: <span className="url-quote">{'http://example.com/'}</span>}),
         target: URLCleanup.ERROR_TARGETS.URL,
       };
+    } else if (isNewOrChangedLink && isExample(link.url)) {
+      error = {
+        message: l(`Example links are not allowed.
+                Did you mean to paste something else?`),
```

Comment:
Indent under Example I guess?

You could also say "{example_url} is just an example. Did you mean to paste something else?" or even "{example_url} is just an example. Please enter the actual link you want to add." But I'm also ok with this, especially since you're blocking other example domains :)

Reply:
Used your last suggestion for commit <https://github.com/metabrainz/musicbrainz-server/pull/2279/commits/49bcd9ad7aa1cdabbd372e9912a4417340c30b53>. Thanks!